apply-templates.sh

```diff
@@ -39,13 +39,11 @@ for version; do
 
 		alpineVer="${suite#alpine}" # "3.12", etc
 		if [ "$suite" != "$alpineVer" ]; then
-			template='Dockerfile-alpine.template'
 			from="alpine:$alpineVer"
 		else
-			template='Dockerfile-debian.template'
 			from="debian:$suite-slim"
 		fi
-		export from
+		export from alpineVer
```

Comment:
I figured since we extracted it, we could use it and not re-extract it via `from`, but so far it was only needed once